src/ert/ensemble_evaluator/config.py

```diff
@@ -95,80 +94,38 @@ def _generate_certificate(
 
 
 class EvaluatorServerConfig:
-    """
-    This class is responsible for identifying a host:port-combo and then provide
-    low-level sockets bound to said combo. The problem is that these sockets may
-    be closed by underlying code, while the EvaluatorServerConfig-instance is
-    still alive and expected to provide a bound low-level socket. Thus we risk
-    that the host:port is hijacked by another process in the meantime.
-
-    To prevent this, we keep a handle to the bound socket and every time
-    a socket is requested we return a duplicate of this. The duplicate will be
-    bound similarly to the handle, but when closed the handle stays open and
-    holds the port.
-
-    In particular, the websocket-server closes the websocket when exiting a
-    context:
-
-       https://github.com/aaugustin/websockets/blob/c439f1d52aafc05064cc11702d1c3014046799b0/src/websockets/legacy/server.py#L890
-
-    and digging into the cpython-implementation of asyncio, we see that causes
-    the asyncio code to also close the underlying socket:
-
-       https://github.com/python/cpython/blob/b34dd58fee707b8044beaf878962a6fa12b304dc/Lib/asyncio/selector_events.py#L607-L611
-
-    """
-
     def __init__(
         self,
         custom_port_range: range | None = None,
         use_token: bool = True,
-        generate_cert: bool = True,
         custom_host: str | None = None,
+        use_ipc_protocol: bool = True,
     ) -> None:
-        self._socket_handle = find_available_socket(
-            custom_range=custom_port_range, custom_host=custom_host
-        )
-        host, port = self._socket_handle.getsockname()
-        self.protocol = "wss" if generate_cert else "ws"
-        self.url = f"{self.protocol}://{host}:{port}"
-        self.client_uri = f"{self.url}/client"
-        self.dispatch_uri = f"{self.url}/dispatch"
-        if generate_cert:
-            cert, key, pw = _generate_certificate(host)
-        else:
-            cert, key, pw = None, None, None
-        self.cert = cert
-        self._key: bytes | None = key
-        self._key_pw = pw
-
-        self.token = _generate_authentication() if use_token else None
```

Comment:
Maybe we can delete this function if it is not used elsewhere

Reply:
I have removed that function and more ....